shared/metrics/src/main/java/io/pravega/shared/metrics/Metric.java

```diff
@@ -5,21 +5,23 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Do not reformat the license header.

Reply:
The formatting tool removed the leading spaces - please confirm indeed we need to add the leading two spaces back?

Reply:
Please do not change it if it's already formatted. You can change your IDE to ignore this.

Reply:
fixed